portfolio/src/main/webapp/script.js

```diff
@@ -76,7 +76,7 @@ function getComments() {
 		const bodyElement = document.body;
         console.log('Comments: ', comments);
         bodyElement.innerHTML += 'Here are all the comments: <br>';
-        for (index in comments) {
+        for (comment of comments) {
     		bodyElement.innerHTML += comments[index] + '<br>';
```

Comment:
With the switch to the `for...of` construct, `index` no longer exists and you should be able to use `comment` directly.
```suggestion
    		bodyElement.innerHTML += comment + '<br>';
```